### ISSUE TITLE
add `rational-post` subcommand to `benchy` + capture benchmark on `master` builds

### ISSUE THREAD

**Issue by laser** [filecoin-project/rust-fil-proofs#846]
### BLOCKERS

Note that this story is blocked on Rational PoSt being merged into rust-fil-proofs master.

### Description

We should be able to use `benchy` to generate and verify a PoSt. To start, we should add a `benchy rational-post` command which accepts a single argument, `size` (the sector size, in KiB, like `benchy zigzag` accepts).

We should measure wall clock and CPU time for `generate_post` and `verify_post` as well as maximum resident set size (RAM).

**Comment by dignifiedquire** [comment]
unblocked, rational post was merged into master

**Comment by dignifiedquire** [comment]
@DrPeterVanNostrand what is the state on of this?

**Comment by DrPeterVanNostrand** [comment]
@dignifiedquire I'm finishing it up now
